src/storage.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+export const addAnswerToStorage=(answer)=>{
```

Comment:
A common folder in project is a `lib` folder, that folder contains all code which are independent from the rest of the code, handy functions. Maybe you can move this file in a directory called 'lib'.